tests/python_package_test/test_plotting.py

```diff
@@ -89,7 +89,7 @@ def test_plot_split_value_histogram(params, breast_cancer_split, train_data):
                                          title='Histogram for feature @index/name@ @feature@',
                                          xlabel='x', ylabel='y', color='r')
     assert isinstance(ax1, matplotlib.axes.Axes)
-    title = 'Histogram for feature name {}'.format(gbm1.booster_.feature_name()[27])
+    title = f'Histogram for feature name {gbm1.booster_.feature_name()[27]}'
```

Comment:
please update your branch to the latest `master`. This change was already made in #4359